golang/main.go

```diff
@@ -1,7 +1,37 @@
 package main
 
-import "fmt"
+import (
+	"fmt"
+	"math"
+)
+
+func task1(x1 float64, x2 float64) {
+	var a1 float64 = 4.1
+	var b1 float64 = 2.7
+	var deltax float64 = 0.7
+	var answersA = []float64{}
```

Comment:
```suggestion
	var answersA = make([]float64, 0, (xc-x1)/deltax)
```
тогда перевыделение памяти будет происходить реже (если вообще будет)